Add descriptions to children docs-Studio Pro How-tos (#5082)
mendix/docs: content/en/docs/howto/logic-business-rules/translatable-validation-messages.md

url: /howto/logic-business-rules/translatable-validation-messages/
category: "Logic & Business Rules"
weight: 140

> description: "Gives an overview of how to use translatable validation messages."

Reviewer: I like "Describes" for consistency with all the others. How about
```suggestion
description: "Describes the use of translatable validation messages."
```

Or you could begin "Describes how to…" for consistency with the other descriptions.

Author: Changed.
